src/services/redisSessionStorage.ts

```diff
@@ -243,6 +274,36 @@ export default class RedisSessionStorage implements ISessionStorage {
     });
   }
 
+  /**
+   * Return a Session for this token.
+   */
+  private loadSessionByWalletToken(
+    token: WalletToken
+  ): Promise<Either<Error, Session>> {
+    return new Promise(resolve => {
```

Comment:
as a shortcut you can either return Promise.resolve(...) (to avoid to type `return resolve())` or make the function async (to avoid to type Promise.resolve())